tests/eth2/beacon/chains/test_chain.py

```diff
@@ -47,7 +47,7 @@ def test_canonical_chain(valid_chain, genesis_slot):
 
     block = genesis_block.copy(
         slot=genesis_block.slot + 1,
-        parent_root=genesis_block.root,
+        previous_block_root=genesis_block.root,
```

Comment:
Logically I think it's better to set to:
```suggestion
        previous_block_root=genesis_block.signed_root,
```
Although they are the same since no genesis block's signature is `EMPTY_SIGNATURE`.